bin/ci-reindex-reviewed-preprints

```diff
@@ -4,6 +4,6 @@ set -e
 
 . bin/ci-start-processes
 
-./bin/console reviewedPreprint:reindex
+./bin/console queue:import ReviewedPreprints
```

Comment:
As we are no longer using the reviewed preprints config we can get rid of the `reviewedPreprint:reindex` command as it duplicates the above command.